INN/umbrella-currentorg
# Internal qa updates

## Changes

This pull request makes the following changes:

- Adds "X (number) of results found:" under search
- Adds space between filters
- Makes expanded category dropdown absolute so it overlays on top of content
- Makes paragraph and details font size = 16 px
- Add "p" 12px top margin
- Makes titles  26px
- Update categories padding to `4px 6px`
- Updates single view and list item padding
- Makes titles `@azure`
- Makes category/tag backgrounds white instead of lavender
- Styles the org type dropdown to match the category dropdown
- Removes images from list view
- Removes background color from categories/tags on single view
- Adds spacing to single view category delimiter
- Adds background color for different "Status" types
- Adds `@azure` left border to single view and active list item

![screencapture-currentorg-test-local-projects-2020-06-04-16_27_40](https://user-images.githubusercontent.com/18353636/83807680-790b5800-a681-11ea-8f08-ca7249b3e067.png)

## Why



For #142

## Testing/Questions

Features that this PR affects:

- The entire project view


Questions that need to be answered before merging:


Steps to test this PR:

1. View the projects page and poke around. Make sure none of the new styles broke anything
